start

```diff
@@ -22,11 +22,13 @@ export PGPORT=5432
 
 QUICKSTART_INITIALIZED=false
 CURRENT_POSTGRES_PID=""
+QUICKSTART_LOG="/quickstart.log"
```

Comment:
Can we write it to `/var/log/`? That's where logs go.

Reply:
yes, adjusted https://github.com/stellar/quickstart/pull/403/commits/304e8fd4ab9411eb4179337becb17e4aed31c73a